modules/update-service-overview.adoc

```diff
@@ -47,6 +47,5 @@ The OpenShift Update Service is composed of an Operator and one or more applicat
 
 [NOTE]
 ====
-During the upgrade process, nodes in the cluster might become temporarily unavailable. The `MachineHealthCheck` might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, remove any `MachineHealthCheck` resource that you have deployed before updating the cluster.
-However, a MachineHealthCheck resource that is deployed by default (such as `machine-api-termination-handler`) cannot be removed and will be recreated.
-====
+During the upgrade process, nodes in the cluster might become temporarily unavailable. The `MachineHealthCheck` might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, pause the `MachineHealthCheck` resource before updating the cluster.
```

Comment:
same here
`..., pause all MachineHealthCheck resources ...`

Reply:
@slintes Accepted and updated in both files.